src/sentry/receivers/rules.py

```diff
@@ -14,7 +14,7 @@
     }
 ]
 DEFAULT_RULE_DATA = {
-    "match": "all",
+    "action_match": "all",
```

Comment:
since this is just a json blob, did you also ensure the UI doesn't have `match` set anywhere for this value? (i tried looking for `match` in typescript code.. but there's a lot 😅)

Reply:
I checked but couldn't find anything, the response serializer converts the field name to  `actionMatch` where we fallback to the default if `action_match` isn't in the JSON
https://github.com/getsentry/sentry/blob/09ee2959be9cdbcb1ebc5c4df2f3f158c6528ce4/src/sentry/api/serializers/models/rule.py#L231